AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/nativead/ManualNativeAdActivity.kt

```diff
@@ -8,10 +8,12 @@ import com.adjust.sdk.AdjustAdRevenue
 import com.adjust.sdk.AdjustConfig
 import com.applovin.enterprise.apps.demoapp.R
 import com.applovin.enterprise.apps.demoapp.ui.BaseAdActivity
+import com.applovin.impl.mediation.model.MediatedNativeAd;
 import com.applovin.mediation.MaxAd
 import com.applovin.mediation.MaxAdRevenueListener
 import com.applovin.mediation.MaxError
 import com.applovin.mediation.nativeAds.MaxNativeAdListener
+import com.applovin.mediation.nativeAds.MaxNativeAd;
```

Comment:
Again it looks like there are imports added here that aren't actually used, we should make the changes consistent across both Activities in Java/Kotlin

Reply:
Thats a good point about keeping the changes consistent. I noticed that there is a line between the imports that start with android. and the ones that start with com. in the Java files, while the Kotlin ones do not have that, should I change this?

Reply:
I would leave it - no need to reformat other files that don't relate to the core PR